src/test/smart-components/user/addUserToGroup.test.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, screen, waitFor } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import AddUserToGroup from '../../../smart-components/user/add-user-to-group/add-user-to-group';
```

Comment:
Just nitpicking, it's more clear to have "local imports" below the ones from external libraries